recipe/meta.yaml

```diff
@@ -21,26 +20,59 @@ requirements:
     - {{ compiler("c") }}
     - {{ compiler("fortran") }}
     - perl
-    - make                       # [not win]
-    - cmake                      # [win]
-    - curl                       # [win]
-    - jom                        # [win]
+    - make  # [not win]
+    - cmake  # [win]
+    - curl  # [win]
+    - jom  # [win]
   host:
   run:
 
-test:
-  commands:
-    - test -f ${PREFIX}/lib/libopenblas.a                            # [unix]
-    - test -f ${PREFIX}/lib/libopenblas${SHLIB_EXT}                  # [unix]
-    - python -c "import ctypes; ctypes.cdll['${PREFIX}/lib/libopenblas${SHLIB_EXT}']"  # [unix]
-    - if not exist %PREFIX%\\Library\\lib\\openblas.lib exit 1       # [win]
-    - if not exist %PREFIX%\\Library\\bin\\openblas.dll exit 1       # [win]
+outputs:
+  - name: libopenblas
+    build:
+      skip: true  # [win and vc<14]
+      run_exports:
+        - {{ pin_subpackage("libopenblas") }}
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("fortran") }}
+    files:
+      - lib/libopenblasp*{{ SHLIB_EXT }}   # [not win]
+      - lib/libopenblas.so.*               # [linux]
+      - lib/libopenblas.*.dylib            # [osx]
+      - Library/bin/*.dll                  # [win]
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libopenblasp-r{{ version }}{{ SHLIB_EXT }}  # [not win]
+        - python -c "import ctypes; ctypes.cdll['${PREFIX}/lib/libopenblasp-r{{ version }}{{ SHLIB_EXT }}']"  # [not win]
+        - if not exist %PREFIX%\\Library\\bin\\openblas.dll exit 1  # [win]
+
+  - name: openblas
+    build:
+      skip: true  # [win and vc<14]
+    files:
+      - include/*.h  # [not win]
+      - lib/*.a  # [not win]
+      - lib/libopenblas${SHLIB_EXT}  # [not win]
+      - lib/pkgconfig/*.pc  # [not win]
+      - lib/cmake/openblas/*  # [not win]
+      - Library/include/*  # [win]
+      - Library/lib/*  # [win]
```

Comment:
This would include unrelated `flang.lib`, right?

Reply:
The files in that folder are:

```
/Library/lib/pkgconfig/openblas.pc
openblas.lib
```

no `flang.lib`. I check the package `openblas-0.3.6-h828a276_2.tar.bz2`.